src/pages/AddScholarship.jsx

```diff
@@ -10,51 +10,44 @@ function AddScholarship() {
   const { t } = useTranslation();
 
   return (
-    <Container maxWidth="md" sx={{ p: 2 }}>
```

Comment:
One thing I'm curious about is if we can keep this outer container while maximizing the screen width usage on medium devices. It looks awkward on huge screens (or if you zoom out):

<img width="1423" alt="Screen Shot 2022-04-25 at 8 57 02 PM" src="https://user-images.githubusercontent.com/8023233/165218014-4f56344e-e4eb-47ac-a382-d29fe6c9794b.png">


Would setting `maxWidth` to `lg` or `xl` address this?

Reply:
maxWdith of 'xl' works.